sdk/daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Blinding.scala

```diff
@@ -125,7 +126,10 @@ object Blinding {
   def partyPackages(
```

Comment:
Please add a ScalaDoc that unambiguously states what is being returned here. So that we don't need to dive into the implementation, next time.